docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,23 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## KPIs and metrics
+
+To ensure effective data ingestion in Azure, these KPIs provide a framework for evaluating key aspects such as completeness and quality.
+
+| **KPI** | **Definition** | **Formula** |
+|----------|-----------|-----|
+| Data completeness | Measures the extent to which all required data fields are present in the dataset and tracks the overall data completeness trend over a specified period.| Percentage of data fields that are complete and the overall data completeness over time. |
+| Data quality | Measures the percentage of successful data quality checks and the total number of data quality checks conducted within a specified period. | Number of data quality checks conducted and the percentage of successful data quality checks . |
+|Data ingestion frequency | Measures how often data is ingested into the system. | Number of data ingestion events per unit of time (daily, weekly, monthly, quarterly, annually). |
+| Data size | Measures the total volume of data ingested into the repository. | Total volume of data ingested into the repository.  |
+| Growth rate | Measure the rate at which the volume of data ingested is increasing over time. | Percentage increase of total data volume in repository per unit of time. |
```

Comment:
This is another good example of where we should add details about why it's important and what to do with it. This may not be obvious to everyone. It may also be useful to include the cost of the storage, compute, and networking costs associated with the ingestion process 🤔 Compute would also be applicable in reporting and not sure if we can differentiate, but something to consider.